common/concurrency/ConcurrentQueue.h

```diff
@@ -60,7 +62,12 @@ template <class Elem, class Queue> class AbstractConcurrentBoundedQueue {
         if (!sorbet::emscripten_build) {
             ret.shouldRetry = elementsLeftToPush.load(std::memory_order_acquire) != 0;
             if (ret.shouldRetry) {
-                sorbet::Timer time(log, "wait_pop_timed");
+                std::unique_ptr<sorbet::Timer> time;
```

Comment:
WDYT about removing the timer from here instead?
The reasoning for keeping it here was that emitting a log line every ~250ms is fine.

Now, I think it's a surprising implication that using unbounded collection removes timers.
WDYT?